openid-sse-framework-1_0.xml

```diff
@@ -887,7 +775,342 @@ Cache-Control: no-store
+            when there is only one Event Stream configured:</t>
+
+            <figure title="Example: Read Stream Configuration Response" anchor="figreadconfigrespnostreamidonestream">
+              <artwork><![CDATA[
+HTTP/1.1 200 OK
+Content-Type: application/json
+Cache-Control: no-store
+
+[
+  {
+    "stream_id": "f67e39a0a4d34d56b3aa1bc4cff0069f",
+    "iss": "https://tr.example.com",
+    "aud": [
+        "http://receiver.example.com/web",
+        "http://receiver.example.com/mobile"
+      ],
+    "delivery": {
+      "delivery_method":
+        "https://schemas.openid.net/secevent/risc/delivery-method/push",
+      "url": "https://receiver.example.com/events"
+    },
+    "events_supported": [
+      "urn:example:secevent:events:type_1",
+      "urn:example:secevent:events:type_2",
+      "urn:example:secevent:events:type_3"
+    ],
+    "events_requested": [
+      "urn:example:secevent:events:type_2",
+      "urn:example:secevent:events:type_3",
+      "urn:example:secevent:events:type_4"
+    ],
+    "events_delivered": [
+      "urn:example:secevent:events:type_2",
+      "urn:example:secevent:events:type_3"
+    ]
+  }
+]
+]]></artwork>
+            </figure>
+
+            <t>The following is a non-normative example response to a request with no "stream_id"
+            when there are no Event Streams configured:</t>
+
+            <figure title="Example: Read Stream Configuration Response" anchor="figreadconfigrespnostreamidnostreams">
+              <artwork><![CDATA[
+HTTP/1.1 200 OK
+Content-Type: application/json
+Cache-Control: no-store
+
+[]
+]]></artwork>
+            </figure>
+
+            <t>Errors are signaled with HTTP status codes as follows:</t>
+
+            <texttable title="Read Stream Configuration Errors" anchor="tabreadconfig">
+              <ttcol align="left">Code</ttcol>
+              <ttcol align="left">Description</ttcol>
+              <c>401</c>
+              <c>if authorization failed or it is missing</c>
+              <c>403</c>
+              <c>if the Event Receiver is not allowed to read the stream configuration</c>
+              <c>404</c>
+              <c>if there is no Event Stream with the given "stream_id" for this Event Receiver</c>
+            </texttable>
+          </section>
+
+          <section anchor="updating-a-streams-configuration"
+            title="Updating a Stream’s Configuration">
+            <t>An Event Receiver updates the current configuration of a stream by making an HTTP
+              PATCH request to the Configuration Endpoint. The PATCH body contains a <xref target="RFC7159">JSON</xref>
+              representation of the stream configuration properties to change. On receiving a valid request the Event
+              Transmitter responds with a <spanx style="verb">200 OK</spanx> response containing a <xref target="RFC7159">JSON</xref>
+              representation of the updated stream configuration in the body.</t>
```

Comment:
should we say "entire updated stream configuration" or is that implied?

Reply:
+1. It will be better to make that explicit.

Reply:
👍